canary/canary.properties

```diff
@@ -4,13 +4,13 @@ lh.canary.topic.creation.enable=true
 lh.canary.workflow.creation.enable=true
 lh.canary.topic.replicas=1
 lh.canary.topic.partitions=1
-
 # Aggregator settings
 lh.canary.aggregator.enable=true
+lh.canary.metronome.worker.enable=true
 
 # Metronome settings
 lh.canary.metronome.enable=true
-lh.canary.metronome.worker.enable=true
 lh.canary.metronome.frequency.ms=1000
-lh.canary.metronome.threads=1
-lh.canary.metronome.runs=1
+lh.canary.metronome.run.threads=1
```

Comment:
When I read this configuration property, I see two things it could configure:
1. The `LHW_NUM_WORKER_THREADS` configuration
2. The number of threads that do `rpc RunWf` in parallel at once.

Which one is it? Let's update the documentation.